Makefile

```diff
@@ -35,7 +37,8 @@ PIP = bin/pip install --no-index --no-dependencies --find-links $(WHEEL_CACHE) -
 RAWJSFILES = $(shell find $(GUISRC)/app -type f -name '*.js' -not -path "*app/assets/javascripts/*")
 BUILT_RAWJSFILES = $(patsubst $(GUISRC)/app/%, $(GUIBUILD)/app/%, $(RAWJSFILES))
 MIN_JS_FILES = $(patsubst %.js, %-min.js, $(BUILT_RAWJSFILES))
-TEMPLATE_FILES := $(shell find $(GUISRC)/app -type f -name "*.handlebars" -or -name "*.partial")
+OLD_TEMPLATE_FILES := $(shell find $(GUISRC)/app -type f -name "*.handlebars" -or -name "*.partial")
+NEW_TEMPLATE_FILES := $(shell find $(GUISRC)/app/components -type f -name "*.hbs")
```

Comment:
Should we add the `*.partial` extension here so we can use partials in the future?